src/redux/immutable/FetchableData.ts

```diff
@@ -19,7 +19,7 @@ const defaultValues: DefaultValues<Shared> = {
 };
 
 export class FetchableDataRecord<R> extends Record<Shared>(defaultValues) {
-  get record() {
+  get getRecord() {
```

Comment:
Why was this renamed ?

Reply:
Because it extends the interface below
```interface Shared {
  record: any;
  error?: string | void;
  isFetching?: boolean;
  lastFetchedAt?: Date;
}```
And it gives the following error
`(getter) FetchableDataRecord<R>.record: R
'record' is defined as a property in class 'Record<Shared> & Readonly<Shared>', but is overridden here in 'FetchableDataRecord<R>' as an accessor.ts(2611)`